dependencies.yaml

```diff
@@ -0,0 +1,103 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: none
+    includes:
+      - build_python
+      - cudatoolkit
+      - develop
+      - py_version
+      - run_python
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  checks:
+    output: none
+    includes:
+      - develop
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
```

Comment:
Do we really want both `rapidsai` and `rapidsai-nightly` here?

Reply:
Yes. The tests require cudf (among other RAPIDS packages), and the pinning of `cudf=23.02` requires `rapidsai-nightly`. For any dependencies on other RAPIDS packages, we want to test against nightlies so we are aware of breakage before release time.